examples/internal_tide.jl

```diff
@@ -149,7 +149,7 @@ function progress(sim)
 end
 
 add_callback!(simulation, progress, name=:progress, IterationInterval(200))
-nothing #hide
+nothing # hide
```

Comment:
```suggestion
nothing #hide
```

Reply:
They’re both valid but I like the version with a space more

Reply:
Or maybe I’m wrong. Did it change?

Reply:
oh OK... not sure.
If you are sure they are valid then go for it! I'm just interpreting the notes